### PR TITLE
fix(select): fix initial values with reactive forms

### LINKED ISSUE #1973: Md-Select start value or active value
#### Bug, feature request, or proposal:
Feature request

#### What is the expected behavior?
The Md-Select-Component should be able to have a start value, which should remain active even after the page has been reloaded. 

After reloading or switching routes I wanna see it like this:
Image

#### What is the current behavior?
We can't set a start value for select. We can click a button to set it, but there is no attribute available to bind it directly with a variable.

#### What are the steps to reproduce?
Look into existing select examples and try to set a startup value. 

#### Which versions of Angular, Material, OS, browsers are affected?
Angular Material latest version (from master)

#### Is there anything else we should know?
I am not sure if I just don't see the way to do it or if it's already planned for the future.

### PR DESCRIPTION
**Pull request by kara**
Closes #1973

r: @jelbourn

### CODE REVIEW

File: src/lib/select/select.ts [hunk 1]
```diff
@@ -175,7 +177,10 @@ export class MdSelect implements AfterContentInit, ControlValueAccessor, OnDestr
    * required to integrate with Angular's core forms API.
    */
   writeValue(value: any): void {
-    if (!this.options) { return; }
+    if (!this.options) {
```
Comment (jelbourn) [inline src/lib/select/select.ts]: Add comment explaining why this is necessary

File: src/lib/select/select.spec.ts [hunk 2]
```diff
@@ -244,8 +244,28 @@ describe('MdSelect', () => {
 
     beforeEach(() => {
       fixture = TestBed.createComponent(BasicSelect);
+    });
+
+    it('should take an initial view value', () => {
```
Comment (jelbourn) [inline src/lib/select/select.spec.ts]: "...with reactive forms" ?

**Comment by qdouble** [comment]
@kara I'm building from master, although setting the initial value works, there seems to be a bug with the page scrolling down to the md-select when the value is set.

**Comment by kara** [comment]
@qdouble That will be fixed when #2054 gets in.

**Comment by qdouble** [comment]
@kara okay great, thanks for the info

**Pull request merged**